README.md

```diff
@@ -1,2 +1,39 @@
-# FilmProject
-With Login and Registration Film website
\ No newline at end of file
+Project Name: Film Info Project
+
+An application used to get Film Info Data with TheMovieDB API, built with React, Redux, JavaScript, and CSS.
+
+Project Status
+This project is currently in development. User can register and log in to their accounts, also can get some data but only partial data.
+
+Installation and Setup Instructions
+Clone down this repository. You will need node and npm installed globally on your machine.
+
+Installation:
+
+npm install
+
+To Run Test Suite:
+
+npm test
```

Comment:
the same here highlight it and every command to be more user friendly and readbale